HACKING.md

```diff
@@ -18,23 +18,52 @@ dnf install -y oci-kvm-hook # if on traditional
 ```
 
 We will use `oc cluster up` to set up a local cluster for testing. To do
-this, simply obtain the OpenShift binary from
-[the latest release](https://github.com/openshift/origin/releases/latest)
-(though note that I'm currently still using the binaries from v3.9, so
-pick that release if you hit issues).
+this, obtain the OpenShift binary from
+[the v3.9.0 release](https://github.com/openshift/origin/releases/tag/v3.9.0).
+`openshift-origin-client-tools` is enough for `oc cluster up`.
+
+If you have not done so already, follow the instructions in the
+[Prerequisites](https://github.com/openshift/origin/blob/v4.0.0-alpha.0/docs/cluster_up_down.md#prerequisites)
+and [Getting Started](https://github.com/openshift/origin/blob/v4.0.0-alpha.0/docs/cluster_up_down.md#getting-started)
+sections to set up your environment and start the Docker daemon.
+
+If on Fedora 29, one extra step is needed. Change the
```

Comment:
Let's modify this slightly to "if on Fedora > `28` and Openshift OKD < `3.11` one extra step is needed...